java-manta-it/src/test/java/com/joyent/manta/config/IntegrationTestConfigContext.java

```diff
@@ -31,12 +31,14 @@
 
     private static String suiteRunId = UUID.randomUUID().toString();
 
+    private static final BaseChainedConfigContext BASE = new StandardConfigContext();
+
```

Comment:
undo since `enableTestEncryption` takes a `SettableConfigContext<T>` and would modify the `static BaseChainedConfigContext`, i.e. *this is bad*